src/cute_shader/cute_shader.cpp

```diff
@@ -520,6 +526,7 @@ void cute_shader_free_result(CF_ShaderCompilerResult result)
 		free((char*)shader_info->image_names[i]);
 	}
 	free(shader_info->image_names);
+	free((int*)shader_info->image_binding_slots);
```

Comment:
So the cast to free is not necessary if it wasn't const in the struct in the first place.